docs/scripts/utils.js

```diff
@@ -50,6 +51,42 @@ export function getImage() {
   return img;
 }
 
+var pieChart = null;
```

Comment:
We use const and var, think it's better to use const and let instead. Bit cleaner

Reply:
I agree with and I wanted to use `let` but, on the top of `utils`, there is `var _seed = Date.now();` therefore I assumed that in utils file the use of `var` is preferable.
Can I change both?

Reply:
I have changed both to **let**